hypha/settings/base.py

```diff
@@ -342,10 +324,7 @@
 
 # Celery settings
 
-if env.str('REDIS_URL', None):
-    CELERY_BROKER_URL = env.str('REDIS_URL')
-else:
-    CELERY_TASK_ALWAYS_EAGER = True
+CELERY_TASK_ALWAYS_EAGER = True
```

Comment:
Unless we are supporting celery, this config should go away as well.

Reply:
Looks like we are using it:

https://github.com/HyphaApp/hypha/blob/1055200594bc7df27bd2471ee141769df92e6d4c/hypha/apply/activity/tasks.py#L5-L50